cmd/manager/main.go

```diff
@@ -146,6 +147,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := contrailcni.Add(mgr, vrouterClusterInfo); err != nil {
```

Comment:
Can you rename the VrouterClusterInfo interface to the ContrailCNIClusterInfo or CNIClusterInfo and remove passing it to the Vrouter controller?  I do not think it is needed there anymore

Reply:
Done